code/modules/cargo/exports/food_and_drink.dm

```diff
@@ -0,0 +1,361 @@
+		/obj/item/reagent_containers/food/snacks/cornchips,
+		/obj/item/reagent_containers/food/snacks/donut/plain,
+		/obj/item/reagent_containers/food/snacks/donut/jelly/plain,
+		/obj/item/reagent_containers/food/snacks/donkpocket,
+		/obj/item/reagent_containers/food/snacks/cookie,
+		/obj/item/reagent_containers/food/snacks/popcorn,
+		/obj/item/reagent_containers/food/snacks/breadslice,	//same reasoning as cake slices, but breads are cheaper.
+		/obj/item/reagent_containers/food/snacks/tofu,
+		/obj/item/reagent_containers/food/snacks/spiderleg,
+		/obj/item/reagent_containers/food/snacks/boiledspiderleg,
+		/obj/item/reagent_containers/food/snacks/spidereggs,
+		/obj/item/reagent_containers/food/snacks/spiderling,
+		/obj/item/reagent_containers/food/snacks/butter,
+		/obj/item/reagent_containers/food/snacks/salami,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/reagent_containers/food/snacks/watermelonslice,
+		/obj/item/reagent_containers/food/snacks/candy_corn,
+		/obj/item/reagent_containers/food/snacks/hugemushroomslice,
+		/obj/item/reagent_containers/food/snacks/badrecipe,
+		/obj/item/reagent_containers/food/snacks/tortilla,
+		/obj/item/reagent_containers/food/snacks/pineappleslice,
+		/obj/item/reagent_containers/food/snacks/tinychocolate,
+		/obj/item/reagent_containers/food/snacks/fortunecookie,
+		/obj/item/reagent_containers/food/snacks/pie/plain,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl,
+		/obj/item/reagent_containers/food/snacks/salad/boiledrice,
+		/obj/item/reagent_containers/food/snacks/twobread,
+		/obj/item/reagent_containers/food/snacks/soup/wish,
+		/obj/item/reagent_containers/food/snacks/fries,
+		/obj/item/reagent_containers/food/snacks/carrotfries,
+		/obj/item/reagent_containers/food/snacks/tatortot,
+		/obj/item/reagent_containers/food/snacks/boiledegg,
+		/obj/item/reagent_containers/food/snacks/onionrings,
+		/obj/item/reagent_containers/food/snacks/yakiimo,
+		/obj/item/reagent_containers/food/snacks/spaghetti/boiledspaghetti,
+		/obj/item/reagent_containers/food/snacks/faggot)
+	include_subtypes = TRUE
+
+//Tier 2: Fast Food.
+//Most typically, 1-2 ingredients from a roundstart kitchen.
+//Cost is pretty decent considering this is what most chefs will wind up cooking for the crew on a good day.
+/datum/export/food/fast
+	cost = 25
+	message = "of average food"
+	export_types = list(/obj/item/reagent_containers/food/snacks/store/bread/plain,
+		/obj/item/reagent_containers/food/snacks/store/bread/meat,
+		/obj/item/reagent_containers/food/snacks/store/bread/tofu,
+		/obj/item/reagent_containers/food/snacks/baguette,
+		/obj/item/reagent_containers/food/snacks/butterbiscuit,
+		/obj/item/reagent_containers/food/snacks/burger/plain,
+		/obj/item/reagent_containers/food/snacks/burger/human,
+		/obj/item/reagent_containers/food/snacks/burger/tofu,
+		/obj/item/reagent_containers/food/snacks/burger/brain,
+		/obj/item/reagent_containers/food/snacks/burger/rat,
```

Comment:
why is a rat burger average food but spaghetti is junk food

Reply:
Because you can make 48 boiled spaghettis without crafting from one dough if you have t4 parts.